ui/.gitignore

```diff
@@ -21,7 +21,6 @@ speed-measure-plugin*.json
 # misc
 /.sass-cache
 /connect.lock
-/coverage/
```

Comment:
@wajda I'm running coverage on my local machine, I don't recommend to remove directory for coverage report from repo